nle/tests/test_nethack.py

```diff
@@ -405,25 +412,37 @@ def test_new_observation_shapes(self):
         assert _pynethack.nethack.NLE_TERM_LI == terminal_shape[0]
         assert _pynethack.nethack.NLE_TERM_CO == terminal_shape[1]
 
-    def test_observations(self):
-        game = nethack.Nethack()
+    def test_observations(self, game):
         game.reset()
 
         tty_chars = game._obs_buffers["tty_chars"]
         tty_colors = game._obs_buffers["tty_colors"]
-        tty_cursor = game._obs_buffers["tty_cursor"]
 
         top_line = "".join(chr(c) for c in tty_chars[0])
         bottom_sub1_line = "".join(chr(c) for c in tty_chars[-2])
         bottom_line = "".join(chr(c) for c in tty_chars[-1])
         assert top_line.startswith(
-            "Hello Agent, welcome to NetHack!  You are a neutral male human Monk."
+            "Hello MonkBot, welcome to NetHack!  You are a neutral male human Monk."
         )
-        assert bottom_sub1_line.startswith("Agent the Candidate")
+        assert bottom_sub1_line.startswith("MonkBot the Candidate")
         assert bottom_line.startswith("Dlvl:1")
 
         for c, font in zip(tty_chars.reshape(-1), tty_colors.reshape(-1)):
             if chr(c) == "@":
                 assert font == 15  # BRIGHT_WHITE
             if chr(c) == " ":
-                assert font == 8  # NO_COLOR
+                assert font == 0  # NO_COLOR
+
+    def test_crop(self, game):
+        game.reset()
+
+        g_chars = game._obs_buffers["chars"].reshape(-1)
```

Comment:
Perhaps a stupid question but did you print these to check if there even is any color here? :)

Reply:
Do you mean - did I check that the values of g_cols and tty_cols aren't all black (0) and white (7)? Yep! Saw some 3's etc...

Reply:
Awesome. Yes that's what I meant.